WikiWatershed/model-my-watershed
# Add Terms of Use, Privacy Policy, Cookie Policy

## Overview

Adds policy information to the app to inform users of how their data will be used.

Connects #3038 

### Demo

![2018-12-05 17 30 43](https://user-images.githubusercontent.com/1430060/49548397-9c4e1f00-f8b3-11e8-93b8-aeb2b3df1ef4.gif)

![2018-12-05 17 31 57](https://user-images.githubusercontent.com/1430060/49548443-bdaf0b00-f8b3-11e8-9170-8de047a61c54.gif)

![image](https://user-images.githubusercontent.com/1430060/49548466-cc95bd80-f8b3-11e8-9b56-b7d1ffe3f52e.png)

![image](https://user-images.githubusercontent.com/1430060/49548473-d3bccb80-f8b3-11e8-8b97-baafe26a271f.png)

![image](https://user-images.githubusercontent.com/1430060/49548492-dcad9d00-f8b3-11e8-9881-f337ae0e08c0.png)

Tagging @jfrankl and @aufdenkampe for visual review.

## Testing Instructions

* Check out this branch and `bundle --debug`
* Go to [:8000/](http://localhost:8000/) and open the About modal
* Ensure you see the three policy modal links in the bottom
* Click each of them to open a modal. Ensure all the links in each modal work correctly.
* Try opening the modals on short screens and ensure that everything is scrollable and viewable.
